packages/web/src/scss/components/Checkbox/_Checkbox.scss

```diff
@@ -11,7 +11,7 @@ $_field-name: 'Checkbox';
 }
 
 .Checkbox__text {
-    margin-left: form-fields-theme.$spacing;
+    margin-left: form-fields-theme.$padding;
```

Comment:
`padding`? 👀 

(Used in almost every file in this PR, even this variable is assigned to the `margin`.)

Reply:
we agreed with @adamkudrna to switch to gap where it makes sense.